backend/cmm.ml

```diff
@@ -203,17 +203,24 @@ type memory_chunk =
   | Onetwentyeight_unaligned
   | Onetwentyeight_aligned
 
-type vector_cast =
-  | Bits128
-
-type scalar_cast =
-  | Float32_as_float
-  | Float_to_int of float_width
+type reinterpret_cast =
+  | Int_of_value
+  | Value_of_int
+  | Float_of_float32
+  | Float32_of_float
+  | Float_of_int64
+  | Int64_of_float
+  | Float32_of_int32
+  | Int32_of_float32
+  | V128_of_v128
```

Comment:
(I think it is not immediately clear
that the purpose of this one is.)

Reply:
Added a comment